autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/deviceauth/AutofillAuthorizationGracePeriod.kt

```diff
@@ -40,6 +40,16 @@ interface AutofillAuthorizationGracePeriod {
      */
     fun recordSuccessfulAuthorization()
 
+    /**
+     * Requests an extended grace period. This may extend the grace period to a longer duration.
+     */
+    fun requestExtendedGracePeriod()
```

Comment:
Can you give me some extra context about what's this extended grace period is for?

Reply:
More context here: https://app.asana.com/0/0/1208489120228733/1208627439305609/f